tests/kuery/test_eql2kql.py

```diff
@@ -51,3 +52,8 @@ def test_ip_checks(self):
         self.validate("dest:192.168.255.255", "dest == '192.168.255.255'")
         self.validate("dest:192.168.0.0/16", "cidrMatch(dest, '192.168.0.0/16')")
         self.validate("dest:192.168.0.0/16", "cidrMatch(dest, '192.168.0.0/16')")
+
+    def test_wildcard_field(self):
+        with eql.parser.elasticsearch_validate_optional_fields:
+            self.validate('field:value-*', 'field : "value-*"')
+            self.validate('field:value-?', 'field : "value-?"')
```

Comment:
lets also add tests to ensure it _only_ applies to wildcards

```suggestion
            self.validate('field:value-?', 'field : "value-?"')
            self.validate('field:"value-*'", 'field == "value-*"')
            self.validate('field:"value-?"', 'field == "value-?"')
```